src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You should link to the CSS file, not the SCSS file. The browser cannot understand SCSS syntax. Make sure to compile your SCSS to CSS and link to the resulting CSS file instead.